spec/models/category_spec.rb

```diff
@@ -18,5 +14,40 @@
       category = Category.create(title: "new_cat")
       expect(category).to_not be_valid
     end
+
+    it "cannot have duplicate names" do
+      Category.create(title: "new_cat", description: "hello")
+      expect { Category.create!(title: "new_cat", description: "hello") }
+      .to raise_error(ActiveRecord::RecordInvalid)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.